docs/extend/extend-apiml/api-mediation-sso.md

```diff
@@ -0,0 +1,63 @@
+# Zowe API Mediation Layer Single-Sign-On Overview
+
+You can extend Zowe and utilize Zowe Single-Sign-On (SSO) provided by Zowe API Mediation Layer (API ML) to enhance system security and improve the user experience. 
+
+This article provides an overview of the API ML single-sign-on feature, the principle participants in the SSO process, and links to detailed Zowe SSO documentation. Zowe Single-Sign-On is based on single user authentication which produces an access token that represents the user in communication with z/OS services accessible through the API Mediation Layer. The access token is issued by the Zowe Authentication and Authorization Service (ZAAS) which is part of API ML. ZAAS issues an access token based on valid z/OS credentials. This token can be validated by any component participating in SSO. 
+
+**Note:** Currently API ML is capable of providing SSO only in a single security domain.
```

Comment:
Suggestion, a little clearer? 

 "Currently, API ML can only provide SSO in a domain that has one security system in place. "

Reply:
Addressed